config/routes.rb

```diff
@@ -18,6 +18,7 @@
     namespace :v0 do
       resources :tokens, only: [:create]
       get 'manifests/search', to: 'manifests#search'
+      post 'manifests/validate', to: 'manifests#validate'
```

Comment:
it's not an object in our db, but it is a resource. 

similar to a `token` for which we to `tokens#create` (there are no tokens in the db)

You could argue that `search` should live in its own `SearchesController` as well so it would fall into a standard routing structure.